pkg/app/piped/cloudprovider/cloudrun/client_test.go

```diff
@@ -57,3 +59,26 @@ func TestManifestToRunService(t *testing.T) {
 	require.NoError(t, err)
 	assert.NotEmpty(t, got)
 }
+
+func TestManifestToRunRevision(t *testing.T) {
+	rm, err := ParseRevisionManifest([]byte(revisionManifest))
+	require.NoError(t, err)
+	require.NotEmpty(t, rm)
+
+	got, err := manifestToRunRevision(rm)
+	require.NoError(t, err)
+	assert.NotEmpty(t, got)
+}
+
+func manifestToRunRevision(rm RevisionManifest) (*run.Revision, error) {
```

Comment:
sorry, finally I can understand what do you want here 🙏  I think we can make it as RevisonManifest method and move it here
https://github.com/pipe-cd/pipecd/blob/master/pkg/app/piped/cloudprovider/cloudrun/revisionmanifest.go
same with `manifestToRunService` used at L28, should move to `ServiceManifest`
https://github.com/pipe-cd/pipecd/blob/master/pkg/app/piped/cloudprovider/cloudrun/servicemanifest.go

Reply:
Exactly, let me apply it.